gridcomps/generic_gridcomps/SimpleLeafGridComp.F90

```diff
@@ -0,0 +1,85 @@
+#include "MAPL_Generic.h"
+
+module mapl3g_SimpleLeafGridComp
```

Comment:
My intent here was for you to copy over SimpleParentGridComp and rename to ConfigurableGridComp.

And there will be only one, so the directory itself should be `gridcomps/configurable`

Reply:
@tclune will the name

Reply:
@tclune, named changed